mrT_test.go

```diff
@@ -26,23 +27,28 @@ func TestMrT(t *testing.T) {
 	defer os.RemoveAll("./testing/")
 
 	if err = m.Txn(func(txn *Txn) (err error) {
-		txn.Put([]byte("greeting"), []byte("hello"))
-		txn.Put([]byte("name"), []byte("world"))
+		if err = txn.Put([]byte("greeting"), []byte("hello")); err != nil {
+			return
+		}
+
+		err = txn.Put([]byte("name"), []byte("world"))
 		return
 	}); err != nil {
-		return
+		t.Fatal(err)
```

Comment:
t.Fatal will prevent defer from running. please either use t.Panic or just something like:
```
import . github.com/PathDNA/testutils
```
...
```
if FailIf(t, err) { return }
```

Reply:
Not true,

```go
func TestDerp(t *testing.T) {
	defer fmt.Println("Hai!")
	t.Fatal("Uhh")
}

/*
panda@archPanda~/go/src/github.com/itsmontoya/mrT: (integrate_cfile) $ go test -run=^TestDerp$
Hai!
--- FAIL: TestDerp (0.00s)
        mrT_test.go:222: Uhh
FAIL
exit status 1
FAIL    github.com/itsmontoya/mrT       0.002s
*/
```

Reply:
hmm when did that change. good catch

Reply:
I appreciate you looking out for issues like that though, good shit :+1: